targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabAuthenticationContext.cs

```diff
@@ -6,7 +6,7 @@ public PlayFabAuthenticationContext()
         {
         }
 
-        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId, string entityId, string entityType) : this()
+        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId, string entityId, string entityType, string telemetryKey = null) : this()
         {
 #if !DISABLE_PLAYFABCLIENT_API
```

Comment:
I wonder if we should create a preprocessor flag for Telemetry Key stuff... I see that there are DISABLE_PLAYFABENTITY_API and DISABLE_PLAYFABCLIENT_API existing flags but I'm not entirely sure how they work and would like to get confirmation if it's okay to add these changes without a corresponding flag. Any thoughts @amccalib, @jasonsandlin ?

Reply:
I don't think we need a preprocessor flag. That's more for enabling and disabling larges classes of the APIs that you, for instance, might not want to appear in your compiled game. It won't matter for telemetry key.